spring-cloud-gcp-pubsub/src/test/java/org/springframework/cloud/gcp/pubsub/integration/inbound/PubSubInboundChannelAdapterTests.java

```diff
@@ -123,10 +123,7 @@ public void testAckModeAutoAck_nacksWhenDownstreamProcessingFails() {
 		when(this.mockMessageChannel.send(any())).thenThrow(new RuntimeException(EXCEPTION_MESSAGE));
 
 		this.adapter.setAckMode(AckMode.AUTO_ACK);
-
-		assertThatThrownBy(() -> {
-			this.adapter.start();
-		}).hasMessageContaining(EXCEPTION_MESSAGE);
+		this.adapter.start();
 
 		verify(mockAcknowledgeableMessage, times(0)).nack();
```

Comment:
Also verify that it does not ack. Maybe even verifyZeroInteractions.

Reply:
Done.